DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupDelegateViewModelTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using DigitalLearningSolutions.Data.Models.DelegateGroups;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class GroupDelegateViewModelTests
+    {
+        [Test]
+        public void GroupDelegateViewModel_populates_expected_values_with_both_names()
+        {
+            // Given
+            var groupDelegate = GroupTestHelper.GetDefaultGroupDelegate(firstName: "Test", lastName: "Name");

```

Comment:
It's clearer and less likely to cause confusing behaviour if you have the tested fields being set here rather than in a separate file. I'd recommend NBuilder instead - just specify the fields you care about and let it do the rest.

Same in the other test.